akka-runtime/src/main/scala/endless/runtime/akka/deploy/AkkaDeployer.scala

```diff
@@ -2,41 +2,43 @@ package endless.runtime.akka.deploy
 
 import akka.actor.typed.{ActorRef, Behavior}
 import akka.cluster.sharding.typed.ShardingEnvelope
-import akka.cluster.sharding.typed.scaladsl.EntityContext
+import akka.cluster.sharding.typed.scaladsl.{ClusterSharding, EntityContext}
 import akka.persistence.typed.scaladsl.EventSourcedBehavior
 import akka.util.Timeout
 import cats.effect.kernel.{Async, Resource}
-import cats.tagless.FunctorK
 import endless.core.entity._
 import endless.core.event.EventApplier
 import endless.core.interpret._
-import endless.core.protocol.{CommandProtocol, EntityIDCodec}
+import endless.core.protocol.{CommandProtocol, CommandSender, EntityIDCodec}
 import endless.runtime.akka.data._
 import AkkaDeployer._
-import endless.runtime.akka.deploy.internal.EventSourcedShardedEntityDeployer
+import endless.runtime.akka.deploy.internal.EventSourcedShardedRepositoryDeployer
 import org.typelevel.log4cats.Logger
 import endless.core.entity.Deployer
+import endless.runtime.akka.ShardingCommandSender
 
 trait AkkaDeployer extends Deployer {
   type DeploymentParameters[F[_], _, S, E] = AkkaDeploymentParameters[F, S, E]
   type Deployment[F[_], RepositoryAlg[_[_]]] = DeployedAkkaRepository[F, RepositoryAlg]
 
-  def deployRepository[F[_]: Async, ID: EntityIDCodec, S, E, Alg[_[_]]: FunctorK, RepositoryAlg[_[
+  override def deployRepository[F[_]: Async, ID: EntityIDCodec, S, E, Alg[_[_]], RepositoryAlg[_[
       _
   ]]](
       repository: RepositoryInterpreter[F, ID, Alg, RepositoryAlg],
-      entity: EntityInterpreter[F, S, E, Alg],
-      effector: EffectorInterpreter[F, S, Alg, RepositoryAlg]
+      behavior: BehaviorInterpreter[F, S, E, Alg],
+      effector: SideEffectInterpreter[F, S, Alg, RepositoryAlg]
```

Comment:
nit: shall we also name this `sideEffect` as for the Durable Entity?

Reply:
yes, thanks 👍